### DISCUSSION TITLE
Rename Content apps for v9

### DISCUSSION THREAD

**Discussion by patrickdemooij9** [umbraco/Umbraco-CMS#10490] [Features and ideas]
### Umbraco version

v9

### Description

Now that we are moving to v9 and seeing that we also changed the name for the different model builder modes, should we also do this change for content apps?

Content apps started out as something just for content. But we now have the ability to also add them to Members and Document Types (and in the future may be to more places). Their name should ideally be updated to reflect those changes. The suggestions in the issue discussed below that I personally agree with most are:

- Context app
- Editor app

There is some more discussion about it here: https://github.com/umbraco/Umbraco-CMS/issues/8863.

This would be a breaking change as it'll also change the formatting in the package.manifest, but this is probably the best timing to do so together with the other changes of v9.

### How can you help?

If this is accepted, I can go through the code and replace all instances to the new naming.

**Comment by nielslyngsoe** [comment]
`Editor app` is a good proposal and gives some context to it ;-)

Whether this detail will create more annoyance for upgrades, compared to the value of it. I don't know. But now the conversation is started... :-)

**Comment by nul800sebastiaan** [comment]
There's an older discussion about this as well, for reference: https://github.com/umbraco/Umbraco-CMS/issues/8863

**Comment by marcemarc** [comment]
The apps are always contextual to the currently edited item, but they are not always for editors?

So Context Apps get my vote.

But does this unveil a different problem regarding the 'naming of things'?

Naming is hard, and hindsight is great when you observe people beginning to interact with the thing you can see how the naming might not be perfect...

Also when you have a thing named as a thing, even if it's not the perfect name, then over time... 'it becomes the name' eg 'Document Type'

... but then I can also remember multiple scenarios when new things have been released and I've thought hmm, is that really the right name?

Content Apps/ContextApps
Pure Live Models
Local and Global Crops
Master/Replica
Media Picker V3

and I think what happens, because naming is hard, and from a dev point of view you are just trying to get the feature to work, and then everyone who has worked upon it internally becomes to know it as the 'content app' feature... it no longer seems weird to them when it is launched, but it does to everyone seeing it for the first time, or seeing it from the 'editor perspective'

... and maybe this is now a different discussion....

But should there be some sort of 'naming panel' for new settings/features/functionality to try and get that input from the community early on, from developers / editors?
